src/handlers.jl

```diff
@@ -190,17 +190,39 @@ end
 docdict(s::AbstractString) = display_dict(Core.eval(Main, helpmode(devnull, s)))
 
 import Base: is_id_char, is_id_start_char
-function get_token(code, pos)
-    # given a string and a cursor position, find substring to request
-    # help on by:
-    #   1) searching backwards, skipping invalid identifier chars
-    #        ... search forward for end of identifier
-    #   2) search backwards to find the biggest identifier (including .)
-    #   3) if nothing found, do return empty string
-    # TODO: detect operators?
 
+function get_previous_token(code, pos, crossed_parentheses)
+    """
```

Comment:
If you want this to be a docstring it should be placed before the function definition, otherwise, keep it as a comment. Right now it is just an unecessary string in the function body.

Reply:
Fixed